### PR TITLE
Orhan_JS2_Week1_Homework

### PR DESCRIPTION
**Pull request by OrhanGoc**

### CODE REVIEW

File: Orhan_JS2_Week1_Homework/apps.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  // }
+
+  function createBookList() {
+    const root = document.getElementById("root");
+    const ul = document.createElement("ul");
+    root.appendChild(ul);
+    const values = Object.keys(bookInfo);
+
+    for (let i = 0; i < values.length; i++) {
+      const li = document.createElement("li");
```
Comment (rohni) [inline Orhan_JS2_Week1_Homework/apps.js]: I would have put this contents of the loop in a function and then call that within the loop, but this works as well.

File: Orhan_JS2_Week1_Homework/apps.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      h2.innerText = bookInfo[values[i]].author;
+      li.appendChild(h2);
+
+      const h3 = document.createElement("h3");
+      h3.innerText = bookInfo[values[i]].language;
+      li.appendChild(h3);
+    }
+  }
+
+  function addImage() {
```
Comment (rohni) [inline Orhan_JS2_Week1_Homework/apps.js]: Excellent.

File: Orhan_JS2_Week1_Homework/apps.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    const values = Object.keys(bookCovers);
+    for (let i = 0; i < values.length; i++) {
+      const imageElem = document.createElement("img");
+      imageElem.setAttribute("src", bookCovers[values[i]]);
+      const liNewId = document.getElementById(values[i]);
+      liNewId.appendChild(imageElem);
+
+      console.log(imageElem);
```
Comment (rohni) [inline Orhan_JS2_Week1_Homework/apps.js]: Once you are done developing/debugging, unless you have a specific reason to leave it, it is a good idea to remove your logging.

File: Orhan_JS2_Week1_Homework/style.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
+
+.book_list ul {
```
Comment (rohni) [inline Orhan_JS2_Week1_Homework/style.css]: Nice use of classes here.

File: Orhan_JS2_Week1_Homework/style.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+}
+.book_list li {
+  float: left;
+  position: relative;
+  list-style: none;
+  color: blue;
+  border: 1px solid black;
+  margin: 10px;
+  padding: 10px;
+  width: 400px;;
```
Comment (rohni) [inline Orhan_JS2_Week1_Homework/style.css]: Hard-coding width in pixels works here, but means that it is less flexible if you are looking at this page with a phone or tablet.

File: Orhan_JS2_Week1_Homework/apps.js (hunk 3 above)
Comment (OrhanGoc) [reply Orhan_JS2_Week1_Homework/apps.js]: I have forgotten to remove logging:))

**Pull request closed without merging**